CESC-submission/paper.tex

```diff
@@ -208,64 +208,61 @@ \section{Impact of Financial Incentives on Service Delivery}
 re-encryption service. \\
 
 
-\noindent{\bf Monetary incentives in the NuCypher network.} Ursulas are rewarded for the re-encryption service by using two sources; the first is the service fees collected from the policy owner Alice, while the second is inflation rewards, i.e., newly minted NU tokens, coming from the NuCypher network. The latter will be provided during the 
-early stage of the network operation to encourage adoption. It is expected that these rewards will disappear as the token cap is reached, similar to other cryptocurrencies in the space.
-
+\noindent{\bf Monetary incentives in the NuCypher network.} Ursulas are rewarded for their re-encryption service by using two sources; the first is the service fees collected from the policy owner Alice, while the second is a subsidy in the form of newly minted NU tokens distributed on a schedule enforced by the NuCypher protocol. The latter will be provided during the early stage of the network operation to encourage service-provider participation and sustain their operations in lieu of a mature fee market.
 
-Currently, the inflation rewards for each round or period are computed for each Ursula based on the size of her stake given that she confirms to be online during this round. Confirming the status of being online is done by calling a 
-simple function in one of the NuCypher contracts, which is like a signal that the calling Ursula is online. Similarly, for fees, Ursula collects part of the Ether locked by Alice during the policy's duration in proportion to the number of periods up to that point in which Ursula has confirmed her online status.
+Currently, the subsidy size in each round or period is computed for each Ursula based on the size of her stake and duration of commitment to the network, and provided that she confirms to be online during this round. Confirming the status of being online is done by calling a simple function in one of the NuCypher contracts, which is like a signal that the calling Ursula is online. Similarly, for fees, Ursula collects part of the Ether locked by Alice during the policy's duration in proportion to the number of periods up to that point in which Ursula has confirmed her online status.
```

Comment:
the subsidy size in each round or period are computed -> the subsidy size in each round or period is computed